lib/ash/actions/create/bulk.ex

```diff
@@ -574,18 +574,34 @@ defmodule Ash.Actions.Create.Bulk do
          action,
          opts,
          lazy_matching_default_values,
-         base,
+         base_changeset,
          argument_names
        ) do
-    base
+    changeset =
+      case input do
+        %Ash.Changeset{} ->
```

Comment:
So, there are some complexities here. A few of them

* the changeset resource must be validated to be the same as the current resource
* the changeset action must be validated to be the same as the current action, or it must be coerced
* if `changeset.__validated_for_action__` is set to the current action, then we can use the changeset as-is. 
* Otherwise we need to treat it as if it's being called with empty params, which actually has some complex issues of its own. We still need to call `Ash.Changeset.handle_params/4`, but how do we handle any existing modifications that have been made? Perhaps if line 631 does two `Map.merge` calls to merge existing changeset attributes and arguments, that will be sufficient.